server/handler/frontend.go

```diff
@@ -91,24 +94,47 @@ func Static(prefix string, c *FilesConfig) http.Handler {
 	return http.StripPrefix(prefix, http.FileServer(http.Dir(dir)))
 }
 
-func getRequires(result *common.Result, githubURL string) map[string][]Membership {
+func getMethods(result *common.Result) map[string][]string {
+	patternInfo := make(map[string][]string)
+	for _, comment := range result.Methods.Comments {
+		patternInfo["Comments"] = append(patternInfo["Comments"], comment)
+	}
+	for _, commentPattern := range result.Methods.CommentPatterns {
+		patternInfo["Comment Patterns"] = append(patternInfo["Comment Patterns"], commentPattern.String())
+	}
+	for _, bodyPattern := range result.Methods.BodyPatterns {
+		patternInfo["Body Patterns"] = append(patternInfo["Body Patterns"], bodyPattern.String())
+	}
+	if *result.Methods.GithubReview {
+		if len(result.Methods.GithubReviewCommentPatterns) > 0 {
+			for _, githubReviewCommentPattern := range result.Methods.GithubReviewCommentPatterns {
+				patternInfo["Github Review Comment Patterns + Github Review Approval"] = append(patternInfo["Github Review Comment Patterns + Github Review Approval"], githubReviewCommentPattern.String())
+			}
+		} else {
+			patternInfo["Github Review State"] = append(patternInfo["Github Review State"], string(result.Methods.GithubReviewState))
+		}
```

Comment:
this is the only part where i feel like the logic slipped into the view a little, but i thought the result would be more helpful for humans to read.